elections/committee-conflict-of-interest.md

```diff
@@ -0,0 +1,47 @@
+# Committee Conflicts of Interest
+
+## Steering Committee and Code of Conduct Committee
+
+At time of writing, the only known conflict of interest for serving on
+committees in the Kubernetes Community exists with the
+[Steering Committee](/committee-steering/README.md) and
+the [Code of Conduct Committee (CoCC)](/committee-code-of-conduct/README.md).
+
+**_Kubernetes Community members cannot serve on both committees simultaneously._**
+
+This conflict may arise in the following scenarios:
+
+- A standing Steering Committee member has accepted a nomination to serve on
+  the Code of Conduct Committee
+- A standing CoCC member has elected to run for a seat on the Steering
+  Committee
+- A Kubernetes Community member has been nominated for candidacy for both
+  committees
+
+Given the uniqueness of the respective committee charters, their memberships'
+position in the community, and the difference in how these committees need to
+adjudicate on community matters, _candidates must be able to declare their
+preference in which committee they would like to serve on, if elected_.
+
+After the candidate is nominated for a position that would conflict and before
+the conclusion of the first of the two elections to complete, a non-conflicted
+Election Officer or in the event that a Steering Committee election is not
+active, a non-conflicted Steering Committee member will reach out to the
+candidate to determine which committee they want to continue with in the event
+that they are elected.
+
+This process should occur as soon as possible after a potential conflict has
+been identified.
+
+If the candidate is elected in the first election to conclude and that
+appointment matches their preference, the candidate will be withdrawn from the
```

Comment:
👋  Hi everyone, long time no see! 

@justaugustus and I were chatting about this PR casually and I had some thoughts so I figured I'd resurrect myself from the dead, zombie-like, to comment on this briefly. (I'll update my affiliation and email in another PR, then go back to sig docs and hang my head in shame) 

The issue with leaving things up to the candidate's preference is that my instinct tells me that most candidates would prefer a Steering seat over a CoCC one. Steering is higher profile, comes with more power/responsibility and generally "looks better". 

Worded as-is I think we stand to run into an issue where folks run both for CoCC and Steering simultaneously in increased numbers and use CoCC as a "fall back" position, so to speak. There's advantages outside of the project (career-wise, i.e.) to holding committee seats, after all. 

So far, to my knowledge, @parispittman and @tpepper are the only ones who have triggered this particular conflict of interest in practice. In both of their cases, they had served the majority of their Code of Conduct terms (in Paris' case, I think 1 year, and in Tim's 1.5 I believe). 

I would suggest that rather than simply leaving it to preference, Steering **does not allow CoCC members to nominate themselves for SC unless they've completed at least 1 year of their CoCC term**. 

This provides the following benefits:
1. Stability for the CoCC: during the last round of elections we (CoCC) had a newly selected CoCC member put themselves up for SC within 3-4 months of joining the CoCC. It created a bit more flux than we like to see on that committee. Keep in mind that while SC is 10 seats iirc, CoCC is only 5 - one person dropping is 20% of the committee. 
2. Demonstration of commitment: People should not be selected to the CoCC if they aren't willing to commit to the work. While CoCC can provide the community visibility required to run a successful SC candidacy, it shouldn't be viewed as a placeholder by candidates. That cheapens the work and cheapens us as a community. If people aren't willing to commit for at least a year, they shouldn't be putting their name in for the CoCC to begin with 
3. Reduction of conflicts of interest: the real issue isn't a CoCC member being elected to the SC, its **people who run for both simultaneously**. Ideally, people should commit to one or the other. 


Anyways, just my two cents! 

New work is going well and has time for me to do some open source splunking so I'll probably be more around from now on ❤️